### DISCUSSION TITLE
Auto leveling issue.

### DISCUSSION THREAD

**Discussion by VitaliyKrym** [mriscoc/Ender3V2S1#1358] [General]
I just install this firmware on my Ender 3 V2 NEO and realized auto leveling is not working correctly. Tramming wizard shows ~0.25mm difference between left and right
![20240921_090557](https://github.com/user-attachments/assets/b41eb5da-9ba4-453d-8153-81405fcae02a)
Following recommendation on screen leaded to nozzle scratch right side of bed. I did manual corner adjustment with same result on screen. I started Auto mesh with this result
![20240921_090828](https://github.com/user-attachments/assets/bf4137d0-fe8c-4b0d-aac1-d9a4724abc4a)
Saved it as Mesh 0.
Anyway I do not see any Z adjustment according to X/Y position during printing.
I know the bed is slightly warped, but not THAT much.
Any ideas?

**Comment by mriscoc** [comment]
The "Calculating relative heights" screen is irrelevant. Check only the final bed tramming wizard screen. Be sure to follow: https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides specially the max and min positions.

**Comment by VitaliyKrym** [comment]
I did everything as in guide. I'm confused. What parameters should be for V2 NEO? Printing area 220x220x250. Bed size 235x235.
![20240921_085938](https://github.com/user-attachments/assets/4f6a4dec-1b0d-4763-8d04-5fb8e59feb0f)

**Reply by mriscoc** [reply]
Correct the bed tramming by following the wizard instructions.
Check if your printer head is colliding at the right side.

**Comment by VitaliyKrym** [comment]
As I mentioned above "Correct the bed tramming by following the wizard instructions." lead to nozzle badly scratching bed at right side :(
I have to manually adjust corners to be able to print. 
Do I need to send
"
G28 ; Home all axes
M420 S1 Z2 ; Use mesh level upto 2 mm "
before print?

**Reply by mriscoc** [reply]
Yes, when the steppers drivers are disabled it is very possible that the head go down, so a new homming is necessary,

**Comment by VitaliyKrym** [comment]
Should I consider bed size 220 x 220 or 235 x 235mm???

**Reply by mriscoc** [reply]
It depends on your real physical limits, check the calibration guide at the Wiki.
